src/main/java/com/wojciechbarwinski/demo/epic_board_games_shop/OrderCronJob.java

```diff
@@ -0,0 +1,51 @@
+package com.wojciechbarwinski.demo.epic_board_games_shop;
+
+import com.wojciechbarwinski.demo.epic_board_games_shop.entities.Order;
+import com.wojciechbarwinski.demo.epic_board_games_shop.entities.OrderStatus;
+import com.wojciechbarwinski.demo.epic_board_games_shop.repositories.OrderRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Component
+@RequiredArgsConstructor
+public class OrderCronJob {
+
+    private final OrderRepository orderRepository;
+
+    @Scheduled(cron = "0 */10 * * * * ") //every 10 minutes
+    public void cancelOrderAfterTenMinutesOnPLACEStage() {
```

Comment:
Nazwa metody nie powinna zawierać częstotliwości - jeśli trzeba będzie ją zmienić i będzie w propertasach to wówczas zmieniając częstotliwość robimy zmianę w jednej linijce w propertasach  i jest cacy :D 

Druga sprawa, że mamy tutaj nazwę wartości z enuma, a więc frazę całą wielkimi literami - to jest błędna konwencja. Nazwa metody powinna wyrażać intencję, a więc w tym wypadku coś w stylu "cancelAbandonedOrders" albo "cancelOrdersNotPaidForTooLong"